src/GraphQLParser.ApiTests/GraphQLParser.approved.txt

```diff
@@ -186,14 +186,12 @@ namespace GraphQLParser.AST
         public GraphQLDirectives() { }
         public override GraphQLParser.AST.ASTNodeKind Kind { get; }
     }
-    public class GraphQLDocument : GraphQLParser.AST.ASTNode, System.IDisposable
+    public class GraphQLDocument : GraphQLParser.AST.ASTNode
     {
         public GraphQLDocument() { }
         public System.Collections.Generic.List<GraphQLParser.AST.ASTNode> Definitions { get; set; }
         public override GraphQLParser.AST.ASTNodeKind Kind { get; }
-        public System.Collections.Generic.List<GraphQLParser.AST.GraphQLComment>? UnattachedComments { get; set; }
-        public void Dispose() { }
-        protected virtual void Dispose(bool disposing) { }
+        public System.Collections.Generic.List<System.Collections.Generic.List<GraphQLParser.AST.GraphQLComment>>? UnattachedComments { get; set; }
```

Comment:
Looks weird, but okay.  Was hoping that with the new AST classes perhaps this would disappear.

Reply:
Or that perhaps the unattached comments just get added to the list with the others...

Reply:
Not a big deal. Let's keep unattached comments as is.